### PR TITLE
Validate SCC exists before admission

### PR DESCRIPTION
**Pull request by concaf**
This commit verifies that SCCs exist on the cluster before admitting TektonConfig or namespace if annotation exists.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Run `make test lint` before submitting a PR
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

```release-note
NONE
```

### CODE REVIEW

File: pkg/reconciler/openshift/namespace/namespace.go [hunk 1]
```diff
@@ -175,6 +176,14 @@ func (ac *reconciler) admissionAllowed(ctx context.Context, req *admissionv1.Adm
 
 	logger.Infof("Trying to admit namespace: %s with SCC: %s", namespaceObject.Name, nsSCC)
 
+	securityClient := common.GetSecurityClient(ctx)
```
Comment (piyush-garg) [inline pkg/reconciler/openshift/namespace/namespace.go:179]: can we use the verifySCCExist function here ?

File: pkg/reconciler/openshift/tektonconfig/extension.go [hunk 2]
```diff
@@ -21,19 +21,20 @@ import (
...
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
+	pkgCommon "github.com/tektoncd/operator/pkg/common"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/openshift/tektonconfig/extension"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	nsV1 "k8s.io/client-go/informers/core/v1"
 	rbacV1 "k8s.io/client-go/informers/rbac/v1"
 	"k8s.io/client-go/kubernetes"
+
```
Comment (piyush-garg) [inline pkg/reconciler/openshift/tektonconfig/extension.go:37]: empty line

File: pkg/reconciler/openshift/tektonconfig/extension.go [hunk 3]
```diff
@@ -21,19 +21,20 @@ import (
...
-
 	mf "github.com/manifestival/manifestival"
 	security "github.com/openshift/client-go/security/clientset/versioned"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
+	pkgCommon "github.com/tektoncd/operator/pkg/common"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/openshift/tektonconfig/extension"
+
```
Comment (piyush-garg) [inline pkg/reconciler/openshift/tektonconfig/extension.go:32]: empty line

File: pkg/reconciler/openshift/namespace/namespace.go (hunk 1 above)
Comment (concaf) [reply pkg/reconciler/openshift/namespace/namespace.go:179]: no, that exists in a separate package (tektonconfig validation) and not in this reconcile - it is small enough boilerplate that i didn't care to port it here too

**Pull request merged**